src/applications/vre/28-8832/components/GetFormHelp.jsx

```diff
@@ -6,9 +6,10 @@ import Telephone, {
 
 const GetFormHelp = () => (
   <p className="help-talk">
-    Need help filling out the form or have questions about eligibility? Please
-    call VA Benefits and Services at{' '}
-    <Telephone contact={CONTACTS.VA_BENEFITS} />.<br />
+    If you have questions or need help filling out this form, please call our
+    MYVA411 main information line at <Telephone contact={CONTACTS.VA_311} /> and{' '}
+    select 0. We’re here 24/7.
```

Comment:
Check out this Slack conversation - https://dsva.slack.com/archives/C5AGLBNRK/p1608065931112100?thread_ts=1608064849.111800&cid=C5AGLBNRK - related to the `24/7` part... not sure if it's really needed, but letting you know.

Reply:
Thank you! updated the PR.